pinpoint-collector/.env

```diff
@@ -6,7 +6,7 @@ SPRING_PROFILES=release
 PINPOINT_COLLECTOR_NAME=pinpoint-collector
 
 #zookeeper information required
-PINPOINT_ZOOKEEPER_ADDRESS=
+PINPOINT_ZOOKEEPER_ADDRESS=zoo1
```

Comment:
`.env` under collector only applies when users use only collector. In this case, zookeeper addresses can vary. So please leave it empty

Reply:
Okay I'll change it.